packages/client/src/components/Leaderboard/index.tsx

```diff
@@ -70,10 +100,13 @@ export function Leaderboard() {
         <TablePagination
           component="div"
           rowsPerPageOptions={[]}
-          count={30}
+          count={Math.min(allLeaders ? allLeaders.length : 0)}
           page={page}
           onPageChange={handleChangePage}
-          rowsPerPage={5}
+          rowsPerPage={Math.min(
+            allLeaders ? allLeaders.length : 0,
```

Comment:
[Предложение]
Можно сократить ка ки выше
```suggestion
            allLeaders?.length ?? 0,
```